src/types.ts

```diff
@@ -12,7 +12,57 @@ export type RequestParameters = OctokitTypes.RequestParameters;
 export type RequestInterface = OctokitTypes.RequestInterface;
 export type Route = OctokitTypes.Route;
 
-export type Token = string;
+/** GitHub App server-to-server token
+ * @see https://docs.github.com/en/developers/apps/authenticating-with-github-apps#authenticating-as-an-installation */
+export type ActionToken = `ghs_${string}`;
+/** GitHub App server-to-server token
+ * @see https://docs.github.com/en/developers/apps/authenticating-with-github-apps#authenticating-as-an-installation */
+export type InstallallationToken = `ghs_${string}`;
```

Comment:
Is there a reason why these are duplicated?

Reply:
I saw there was a "or" in the example `ghs_InstallallationOrActionToken00000000` and I thought it might possible that they could differ in the future